blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxWriter.java

```diff
@@ -209,10 +209,10 @@ public void writeMetadataValue(Locator locator, String metaKey, String metaValue
         }
     }
 
-    public void writeExcessEnumMetric(Locator locator, Long timestamp) throws ConnectionException {
+    public void writeExcessEnumMetric(Locator locator) throws ConnectionException {
```

Comment:
George and me decided that a timestamp value is not needed for writing to the "bad" enum metrics cassandra CF (metrics_excess_enums).  So I removed the param here.